test/jdk/sun/security/pkcs12/KeytoolOpensslInteropTest.java

```diff
@@ -450,4 +577,57 @@ private static void check(
     static OutputAnalyzer keytool(String s) throws Throwable {
         return SecurityTools.keytool(s);
     }
-}
+
```

Comment:
I feel the lines below should goes to a test library.

Reply:
Moved this part to test/lib/jdk/test/lib/artifacts/OpensslArtifactFetcher.java